content/javascript/concepts/storage/terms/length/length.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: '.length'
+Description: 'The length property returns the number of items in an array or the number of characters in a string.'
```

Comment:
```suggestion
Description: 'Returns the number of items in an array or the number of characters in a string.'
```

Reply:
@yangc95 This change is now implemented. Please take a look. Thanks